samples/build.gradle

```diff
@@ -31,9 +42,9 @@ dependencies {
 
     implementation 'com.datastax.oss:java-driver-query-builder:4.3.1'
 
-    implementation 'org.slf4j:slf4j-api:2.0.0-alpha1'
-    implementation 'org.slf4j:slf4j-log4j12:2.0.0-alpha1'
+    implementation 'com.genesyslab.webme.commons:wcc-es-index:9.1.002.00'
```

Comment:
There is no need to use es-index on client side, the sample should only rely on datastax driver and related log libs.

Reply:
yes of course, it was to test retrieving package from github. Will remove it from sample